content/docs/user-guide/gems/reference/terrain/layer_spawner.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Layer Spawner component
+linktitle: LayerSpawner
+description: ' Open 3D Engine (O3DE) Terrain Layer Spawner reference. '
+weight: 100
+---
+
+{{< preview-migrated >}}
+
+The **Layer Spawner** component defines the bounds of a terrain layer, and enables the ordering of multiple layers.  
+The priority is controlled firstly by assigning a layer, either **Foregound** or **Background**, then by using the **Priority** setting.  
```

Comment:
We should probably mention that Foreground is a higher priority than Background, and for Priority, we should mention that higher values represent a higher priority within that layer.

Reply:
This is stated in the Properties section, but I'll add it here as well.